arquillian-environment-reporter/arquillian-environment-reporter-impl/src/main/java/org/arquillian/environment/reporter/impl/TestRunnerDetector.java

```diff
@@ -0,0 +1,60 @@
+package org.arquillian.environment.reporter.impl;
+
+public class TestRunnerDetector {
+
+    private TestRunnerDetector() {
+    }
+
+    public static TestRunner detect() {
+        return detect(Thread.currentThread().getStackTrace());
+    }
+
+    static TestRunner detect(StackTraceElement[] stackTrace) {
```

Comment:
Just wondering what do we benefit for making it a static methods  - maybe regular object would give us slightly better design?